src/discussions/posts/post/PostFooter.jsx

```diff
@@ -53,7 +53,7 @@ function PostFooter({
             />
           )}
       </OverlayTrigger>
-      {preview
+      {preview && post.commentCount > 1
```

Comment:
I feel the condition should be `post.commentCount > 0`?

Reply:
> 
Backend send `post.commentCount` as `1` if the post has no comment.

Reply:
Backend send `post.commentCount` as `1` if the post has no comment.

Reply:
according to adeel the count is coming 1 from backend if there is no comment. it should be fixed from backend no.

Reply:
It was decided in the meeting that the appropriate way would be to change condition to `post.commentCount > 1` instead of changing from backend.

Reply:
okay, I was unaware of that discussion. however, I would like to discuss why we finalized that but approving it for now.